templates/registration/signup.html

```diff
@@ -14,7 +14,9 @@
                     <div class="px-3 py-5">
                         <div class="row">
                             <div class="col col-lg-6 offset-lg-3">
-                                <img src="{% static 'images/scaleout_black.png' %}" alt="Studio Logo" style="max-width: 100%; height: auto;">
```

Comment:
It feels a bit empty, but perhaps we can add some logo later?

Reply:
I thought of adding logos as it was done by scaleout, but in my opinion, there would be too much visual elements with additional logo